src/lib/styles/app.less

```diff
@@ -0,0 +1,57 @@
+/*
+ * Fonts
+ */
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,300,400,700);
+
```

Comment:
this stuff came from https://github.com/PRX/publish.prx.org/pull/411/files#diff-7d2c77ab7cf4b4554453158f74957bedL37